### DISCUSSION TITLE
Provide a shortcut key for import from clipboard

### DISCUSSION THREAD

**Discussion by badihi** [Kong/insomnia#3766] [Ideas]
In older version it was possible to paste a curl command inside the address bar to import it. Now this feature is removed for some reason and a button inside Import/Export dialog is placed to use instead of that.
So for lazy people like me with a huge number of requests to import every day, it became a little harder after that change.

So I want to ask, please add a shortcut key to import from clipboard instantly without needing to opening that dialog.

That would be so much appreciated.

**Comment by develohpanda** [comment]
Thanks for reporting! Pasting in the address bar to import cURL wasn't intentionally removed, so this is definitely a bug to be fixed.

Related #3002

**Comment by badihi** [comment]
So this is not intentional. I'll be happy to have it back.

However I still insist on the feature that I requested, as it will so much easier to use a shortcut key to create a new request and import the curl into it, instead of creating a new request and pasting the command inside the address bar.

**Comment by wdawson** [comment]
Looks like this bug is resolved (I'm unable to reproduce in 2021.4.0), so I'll be converting this back to a feature request and moving to discussions.
